### DISCUSSION TITLE
Metrics For Tweets

### DISCUSSION THREAD

**Discussion by RyanKramer** [tweepy/tweepy#1605] [Questions] [answered]
Hello,

Would it be possible to include the metrics for the tweet?

I know we have "like" and "retweet count" but I would also like to get these attributes -

"non_public_metrics": {
      "impression_count": 956,
      "url_link_clicks": 9,
       "user_profile_clicks": 34
    },
    "organic_metrics": {
      "impression_count": 956,
      "like_count": 49,
      "reply_count": 2,
      "retweet_count": 9,
      "url_link_clicks": 9,
       "user_profile_clicks": 34

These are the under the metrics - https://developer.twitter.com/en/docs/twitter-api/metrics

Ryan

**Comment by Harmon758** [comment, marked as answer]
These are part of Twitter API v2, support for which is already in the development version on the master branch and will be released as part of Tweepy v4.0. See #1472 for more information.
